data/api/client-server/pushrules.yaml

```diff
@@ -16,228 +16,214 @@ info:
   title: Matrix Client-Server Push Rules API
   version: 1.0.0
 paths:
-  /pushrules/:
+  /pushrules/global/:
```

Comment:
This removes the `GET /_matrix/client/v3/pushrules/` API, which would need an MSC - clients rely on it existing. Unfortunately I think we need to keep both `GET /_matrix/client/v3/pushrules/` and `GET /_matrix/client/v3/pushrules/global` for now.

Reply:
You're correct. I thought I tested this and the endpoint wasn't actually supported, I think I forgot the trailing slash. :sad:

Reply:
I think it is useful to split this out to a separate endpoint still, although copying most of the example over is unfortunate.